### ISSUE TITLE
Are you using an API to get thouse good news?I need something like that

### ISSUE THREAD

**Issue by RenatoCesarF** [wel-shy/good-news-api#2]

**Comment by wel-shy** [comment]
Yeah the news headlines are fetched using the API from [News API](https://newsapi.org/). These are standard news headlines and then I'm using a machine learning algorithm to determine whether they are positive or not. This is the [python job](https://github.com/gd-nws/classifier/blob/master/classifier.py).
